source/sec_community_communication.ptx

```diff
@@ -75,7 +75,7 @@
                       </statement>
                     </definition>
                       <p>
-                      After an issue is posted, discussions on the issue typically occur and are sometimes prolonged. These discussions are most conveniently attached to the issue itself to facilitate the tracking of the discussion, which is particularly important in larger projects. When an issue is posted, it can be discussed by the community to decide on how to proceed with it. The issue can be categorized and linked to other discussions, projects, and engineering work as well. You are definitely advised to read all unresolved issues before posting so you are not duplicating another existing issue. 
+                      After an issue is posted, discussions on the issue typically occur and are sometimes prolonged. These discussions are most conveniently attached to the issue itself to facilitate the tracking of the discussion, which is particularly important in larger projects. When an issue is posted, it can be discussed by the community to decide on how to proceed with it. The issue can be categorized and linked to other discussions, projects, and engineering work as well. You are definitely advised to read or search through all unresolved issues before posting so you are not duplicating an existing issue. 
```

Comment:
The suggestion for "search" here is because bigger projects often have way too many open issues to read through them all. [Nixpkgs](https://github.com/NixOS/nixpkgs/issues) for example has 6K+ open issues at the moment of writing.